types/beacon_types.go

```diff
@@ -94,6 +101,77 @@ type Duty struct {
 	ValidatorSyncCommitteeIndices []uint64 `ssz-max:"13"`
 }
 
+func MapDutyToRunnerRole(dutyRole BeaconRole) RunnerRole {
+	switch dutyRole {
+	case BNRoleAttester, BNRoleSyncCommittee:
+		return RoleCommittee
+	case BNRoleProposer:
+		return RoleProposer
+	case BNRoleAggregator:
+		return RoleAggregator
+	case BNRoleSyncCommitteeContribution:
+		return RoleSyncCommitteeContribution
+	case BNRoleValidatorRegistration:
+		return RoleValidatorRegistration
+	case BNRoleVoluntaryExit:
+		return RoleVoluntaryExit
+	}
+	return RoleUnknown
+}
+
+type BeaconVote struct {
```

Comment:
So after conversation:
`Spec.DataVersion` is unneeded since we don't put eth ssz data anymore.
Whether we need to have `Duty` is subjective like I said... In the meanwhile we don't need it...

So @MatheusFranco99 fix this like @alonmuroch asked in the above comment
Thanks!

Reply:
ConsensusData -> ValidatorData